docs/src/userguide/extension_types.rst

```diff
@@ -545,6 +603,13 @@ usually set on an extension type using a decorator:
             class Parrot:
                def done(self): pass
 
+            @cython.cclass
+            class Lizzard:
+
+               @cython.final
+               @cython.cfunc
+               def done(self): pass
```

Comment:
I'd probably give the function a different name. Giving it the same name as for `Parrot` kind of makes me think this is trying to demonstrate something to do with inheritance.

Reply:
done